Group pages

b-m-f opened the pull request:
**Depends on** https://github.com/canonical-web-and-design/www.ubuntu.com/pull/5071
## Done
- Updates to blog module 1.5.1 https://github.com/canonical-web-and-design/canonicalwebteam.blog/pull/40 
- Create a dummy template for `cloud-and-server` (see blog module for more info on which groups are available)

## QA

**Template are in an early state, and styling was not part of this PR**

- `./run`
- go to `http://localhost:8001/blog/cloud-and-server` and make sure only articles from that group are shown
- go to `http://localhost:8001/blog/cloud-and-server?category=case-studies` and make sure only articles from that group+category are shown



## Issue / Card

Fixes https://github.com/canonical-web-and-design/base-squad/issues/455
Fixes https://github.com/canonical-web-and-design/base-squad/issues/456

webteam-app commented:
Starting demo at: https://www-ubuntu-com-canonical-web-and-design-pr-5087.run.demo.haus/

anthonydillon commented:
#5071 has merged can this one be unblocked?

anthonydillon commented:
@b-m-f Few questions:
- Shouldn't this target a feature branch until all templates are approve?
- What is the scope of this branch? Template is basic and missing page title, etc.
- The issue in Fixes is already closed. Is there an issue for this?

b-m-f commented:
@anthonydillon :
- Definetely! I have switched the base, thanks.
- This branch just adds the necessary templates to route to group pages using the new blog-module@1.5.1 .Frontend work can then be done on these templates (doing one takes some work, the others will then be easy)
- I think this happened when closing the blog module. These 2 PRs both targeted that ticket. I have added another related ticket as well.

anthonydillon commented:
@b-m-f Thanks, then this one doesn't need a front end review :ok_hand:

nottrobin commented on requirements.txt:
```diff
@@ -1,10 +1,10 @@
 Django==2.2
 django-prometheus==1.0.15
-canonicalwebteam.blog==1.5.0
+git+https://github.com/b-m-f/blog-extension@group-pages#egg=canonicalwebteam.blog
```
Is there now a new version that this could be pointed at?

b-m-f replied:
Have updated it to `1.5.1`. Thank you.

nottrobin commented on requirements.txt:
```diff
@@ -1,10 +1,10 @@
 canonicalwebteam.django_views==1.3.4
 canonicalwebteam.get-feeds==0.2.4
-canonicalwebteam.http==0.1.6
+canonicalwebteam.http==1.0.1
```
Try running the site with `./run --env SEARCH_API_KEY=fakekey` - you'll see that this version of `http` causes an error.

I think `webapp/views.py` line 25 just needs changing to use `fallback_cache_duration`.

b-m-f replied:
Great catch. The solution worked!

jpmartinspt commented:
I see articles from other groups in all different group pages.

jpmartinspt commented:
The difference is just in the cards title after all, can probably be sorted at the template level.
